service/patch.go

```diff
@@ -171,6 +172,11 @@ func (uis *UIServer) schedulePatch(w http.ResponseWriter, r *http.Request) {
 			VersionId string `json:"version"`
 		}{projCtx.Version.Id})
 	} else {
+		githubOauthToken, err := evergreen.GetEnvironment().Settings().GetGithubOauthToken()
```

Comment:
settings object is a field on the UI server

Reply:
done